adb_graphics/specs.py

```diff
@@ -320,10 +320,11 @@ def smoke_colors(self) -> np.ndarray:
 
         ''' Default color map for smoke plots. '''
 
-        blues = cm.get_cmap('Blues', 6)(range(5))
+        white = cm.get_cmap('Greys', 2)([0])
+        blues = cm.get_cmap('Blues', 15)([2,5,8,11])
```

Comment:
To leave the blues just as they were before as Ravan requested, you should be able to leave it with the more straightforward:

```
blues = cm.get_cmap('Blues', 6)(range(1, 5))
```